modules/apps/commerce/commerce-service/src/main/resources/com/liferay/commerce/internal/batch/01-object-definition.batch-engine-data.json

```diff
@@ -98,7 +868,7 @@
 			"portlet": false,
 			"scope": "company",
 			"status": {
-				"code": 2
```

Comment:
what is this change about?

Reply:
> what is this change about?

Previously, the creation of objects was split into multiple files. In the first file, the object was created in the Draft (2) state and then the status was changed to Approved (0) in the last file.